README.md

```diff
@@ -17,45 +8,40 @@ Scanner for vulnerabilities in container images, file systems, and Git repositor
 [![GitHub All Releases][github-all-releases-img]][release]
 ![Docker Pulls][docker-pulls]
 
+[📖 Documentation](docs)
+</div>
 
-# Abstract
-`Trivy` (`tri` pronounced like **tri**gger, `vy` pronounced like en**vy**) is a simple and comprehensive scanner for vulnerabilities in container images, file systems, and Git repositories, as well as for configuration issues.
-`Trivy` detects vulnerabilities of OS packages (Alpine, RHEL, CentOS, etc.) and language-specific packages (Bundler, Composer, npm, yarn, etc.).
-In addition, `Trivy` scans Infrastructure as Code (IaC) files such as Terraform, Dockerfile and Kubernetes, to detect potential configuration issues that expose your deployments to the risk of attack.
-`Trivy` also scans hardcoded secrets like passwords, API keys and tokens.
-`Trivy` is easy to use. Just install the binary and you're ready to scan.
+Trivy is the world’s most popular open source security scanner. It is reliable, fast, extremely easy to use, and it works wherever you need it.
 
-<figure style="text-aligh: center">
-  <img src="docs/imgs/overview.png" width="800" alt="Trivy Overview">
-</figure>
+Trivy has different *scanners* that look for different security issues, and different *targets* where it can find those issues.
 
-### Demo: Vulnerability Detection (Container Image)
-<figure style="text-aligh: center">
-  <img src="docs/imgs/vuln-demo.gif" width="1000" alt="Vulnerability Detection">
-</figure>
+Targets:
+- Container Image
+- Filesystem
+- Git repository (remote)
+- Kubernetes cluster or resource
```

Comment:
Could we add here:
- IaC configuration files (such as Dockerfile, Terraform)

Reply:
this is the list of targets. did you mean to add it to the list of scanners? it's already there